src/markovevaluator.cpp

```diff
@@ -51,13 +51,19 @@ Reaction &MarkovEvaluator::GetNextReaction() {
 // [region]
 NetworkState MarkovEvaluator::GetNextNetworkState() {
 	CalculateProbs();
+	if (mState.time > timeThreshold && timeThreshold != 0) {
+		finished = true;
+	}
 	if (finished) {
 		return mState;
 	}
 	Reaction &reaction = GetNextReaction();
 	mState = mState - reaction.reactants;
 	mState = mState + reaction.products;
 	mState.time = currentTime;
+	if (timeThreshold == 0 && print == true) {
```

Comment:
Hvorfor skal timeThreshold være 0 før at den printer? Det virker som to settings der ikke burde have noget med hinanden at gøre